main/main.go

```diff
@@ -140,6 +142,12 @@ var (
 			Value:  "",
 			EnvVar: "DATADOG_TAGS",
 		}),
+		altsrc.NewBoolFlag(cli.BoolFlag{
+			Name:   config.VAR_NAME_DD_PROFILER,
+			Usage:  "enables datadog continuous profiling",
+			EnvVar: "DATADOG_PROFILER",
+			Hidden: true,
+		}),
```

Comment:
if the env value isn't present in the container, what will this default to?

Reply:
False. It has to be set to `true` if present for it to be enabled. So `DATADOG_PROFILER=false` works as you would expect.